actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -901,6 +916,15 @@ private void setStateValidation(
       userBooleanMap.put(
           JsonKey.STATE_VALIDATED, (boolean) userDbRecord.get(JsonKey.STATE_VALIDATED));
     }
+    // adding in release-3.0.0
```

Comment:
Without email, phone, setting these fields doesnt make sense. If there is any failure due to it, lets fix that.

Reply:
@indrajra email and phone are part of it.
This is suggested in the [ticket](https://project-sunbird.atlassian.net/browse/SH-135)